src/main/windows/windowManager.ts

```diff
@@ -465,15 +466,21 @@ export function selectNextTab() {
     }
 
     const currentTeamTabs = status.config?.teams.find((team) => team.name === currentView.tab.server.name)?.tabs;
+    const filteredTabs = currentTeamTabs?.filter((tab) => !tab.isClosed);
     const currentTab = currentTeamTabs?.find((tab) => tab.name === currentView.tab.type);
-    if (!currentTeamTabs || !currentTab) {
+    if (!currentTeamTabs || !currentTab || !filteredTabs) {
         return;
     }
 
-    const currentOrder = currentTab.order;
-    const nextOrder = ((currentOrder + 1) % currentTeamTabs.length);
-    const nextIndex = currentTeamTabs.findIndex((tab) => tab.order === nextOrder);
-    const newTab = currentTeamTabs[nextIndex];
+    let currentOrder = currentTab.order;
+    let nextIndex = -1;
+    while (nextIndex === -1) {
+        const nextOrder = ((currentOrder + 1) % currentTeamTabs.length);
+        nextIndex = filteredTabs.findIndex((tab) => tab.order === nextOrder);
+        currentOrder = nextOrder;
+    }
```

Comment:
why do we need a while structure here? shouldn't this just run once?

Reply:
`filteredTabs` is the list of open tabs. If `nextOrder` would normally pick a tab that's closed, `findIndex` would return -1 and it would error out. We want to make sure it's always picking the next tab.